src/gui/mainwindow.cpp

```diff
@@ -695,6 +697,25 @@ void MainWindow::displayRSSTab(bool enable)
     }
 }
 
+void MainWindow::showFilterContextMenu(const QPoint &)
+{
+    Preferences *const pref = Preferences::instance();
+
+    QMenu *menu = m_searchFilter->createStandardContextMenu();
+    menu->addSeparator();
+    QAction *useRegexAct = new QAction(tr("Use regular expressions"), menu);
+    useRegexAct->setCheckable(true);
+    bool useRegexAsFilteringPattern = pref->getRegexAsFilteringPattern();
```

Comment:
`const bool`

Reply:
I inlined pref->getRegexAsFilteringPattern() because it's now used only once.